### ISSUE TITLE
How do I get label using torchtext?

### ISSUE THREAD

**Issue by oya163** [pytorch/text#483]
For example, if I have training set for binary classification with labels/classes [0, 1]. Is there anyway that I can get these classes directly using torchtext?

**Comment by manuelsh** [comment]
Use a field for the label. See the fourth example in:

https://github.com/pytorch/text/blob/master/README.rst

or look to `Fields`.

**Comment by zhangguanheng66** [comment]
Thanks for answering questions.

**Comment by oya163** [comment]
After buidling vocab for LABEL field
`LABEL.build_vocab(train_data)`

Simply print out the list of labels like this
`print(LABEL.vocab.itos)`
